project/Build.scala

```diff
@@ -81,7 +81,7 @@ object Build {
    *  set to 3.1.3. If it is going to be 3.1.0, it must be set to the latest
    *  3.0.x release.
    */
-  val previousDottyVersion = "3.0.2"
+  val previousDottyVersion = "3.1.0-RC1"
```

Comment:
We need to wait until we have `3.1.0` to update this one.

Reply:
We will probably have to keep the MiMaFilters for now.

Reply:
But then we would need to add lots of exceptions to MiMa, as it wants to check for both forward and backward compatibility between `3.0.2` and what is currently on master. It works like that because `3.1.1` is considered as a patch release and then it needs release from the same versioning line to be compared to.

Reply:
I don't remember exactly why, but last time I was told that we should do it like this. As it is written in the documentation above. @smarter I believe you where the one that told me that. Is that true?

Reply:
Ah, ok.  So I think the change do to is keep this to 3.0.2 but change baseVersion to 3.1.0-RC2 (or something 3.1.0.*).  That way the logic in `compatMode` won't start enforcing bidirectional checking.